charts/pega/templates/_pega-deployment.tpl

```diff
@@ -113,7 +113,7 @@ spec:
           {{- if .node.cpuLimit }}
             cpu: "{{ .node.cpuLimit }}"
           {{- else }}
-            cpu: 2
+            cpu: 4
```

Comment:
I know this is specifically about the max CPU, but should we be increasing the 'requests' default value too?  If Kubernetes decides to overload a node due to a low request value, we will never get anywhere near the limit.  I worry a bit that we are increasing the desired amount but not the minimum.

What is a reasonable minimum CPU amount for Pega?  1vCPU? 2? More?

Reply:
It seems to be a problem for me. However, I don't have exact data to support what would be reasonable minimum CPU.
But 200m seems to be very less. @Mandeep-pega do you have any concerns increasing this limit.

Reply:
I agree. I was having a wrong understanding about cpu requests. How about a default minimum request of 2 for Pega nodes? 50% of limit should be good enough value.